theano/gpuarray/dnn.py

```diff
@@ -1859,13 +1859,10 @@ def local_gpua_pool_dnn_alternative(op, ctx_name, inputs, outputs):
         raise_no_cudnn()
     if not op.ignore_border:
         return
-    img, = inputs
+    img, ws, stride, pad = inputs
```

Comment:
why did you started to rename ds to ws everywhere? We didn't discuss this interface change. I'm not sure it is good. Currently we use ds and ws, and I don't think mixing them is good. So we should decide which one to keep.

This is at many places.